### PR TITLE
Tweak: Remove 'kit by plan' filtering [ED-9138]

### PR DESCRIPTION
**Pull request by ranshoshani34**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:
Removed 'Kit by plan' filtering from kit library
*

## Description
An explanation of what is done in this PR
Removed 'Kit by plan' filtering from the kit library, and the badges indicating plans.
*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by mark-elementor — DISMISSED** [review]

File: app/modules/kit-library/assets/js/components/kit-list-item.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import Badge from './badge';
 import FavoritesActions from '../components/favorites-actions';
+import Badge from "./badge";
```
Comment (ManorHazaz) [inline app/modules/kit-library/assets/js/components/kit-list-item.js:2]: Fix lint

File: app/modules/kit-library/assets/js/components/kit-list-item.js [hunk 2]
```diff
@@ -46,15 +46,13 @@ const KitListItem = ( props ) => {
...
-					{
-						subscriptionPlan?.label &&
-							<Badge
-								variant="sm"
-								className="e-kit-library__kit-item-subscription-plan-badge"
-								style={ { backgroundColor: subscriptionPlan.color } }
-							>
-								{ subscriptionPlan.label }
-							</Badge>
+					{ ! elementorAppConfig.hasPro &&	subscriptionPlan?.label && <Badge
```
Comment (ManorHazaz) [inline app/modules/kit-library/assets/js/components/kit-list-item.js:49]: Span?

File: package.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
         "scripts:watch": "grunt watch_scripts",
         "styles": "grunt styles",
         "styles:watch": "grunt watch_styles",
-        "lint:js": "eslint ."
+        "lint:js": "eslint --fix ."
```
Comment (ManorHazaz) [inline package.json:26]: Nice

File: tests/phpunit/elementor/app/kit-library/data/taxonomies/test-controller.php [hunk 4]
```diff
@@ -63,7 +63,7 @@ public function test_get_items() {
 		$subscription_plans = Plugin::$instance->common->get_component( 'connect' )->get_subscription_plans();
 
 		$this->assertArrayHasKey( 'data', $result );
-		$this->assertCount( 8, $result['data'] );
+		$this->assertCount( 5, $result['data'] );
```
Comment (ManorHazaz) [inline tests/phpunit/elementor/app/kit-library/data/taxonomies/test-controller.php:66]: FIx test

File: assets/dev/js/editor/components/template-library/views/library-layout.js [hunk 5]
```diff
@@ -43,7 +43,7 @@ module.exports = elementorModules.common.views.modal.Layout.extend( {
 				return accessLevel >= templateData.accessLevel;
 			} );
 
-		const subscriptionPlan = subscriptionPlans[ closestAccessLevel ];
+		const subscriptionPlan = subscriptionPlans[ closestAccessLevel ] ?? subscriptionPlans[ subscriptionPlans.length - 1 ];
```
Comment (ManorHazaz) [inline assets/dev/js/editor/components/template-library/views/library-layout.js:46]: FIx it from closestAccessLevel.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by nivnoimanelementor — APPROVED** [review]

**Pull request merged**